litejpeg/core/huffman/huffmancore.py

```diff
@@ -0,0 +1,479 @@
+"""
+HuffmanEncoder Module:
+----------------------
+The module takes an input in the form of matrix containing 64 blocks with
+20 bits each and give an output of variable number each having a bit_length
+of 8 bits.
```

Comment:
Put newlines between each of these paragraphs.

Reply:
Done